composer.json

```diff
@@ -33,7 +33,7 @@
         "ext-openssl": "*",
         "ext-session": "*",
         "ezyang/htmlpurifier": "^4.17",
-        "henrique-borba/php-sieve-manager": "^1.0.2",
+        "henrique-borba/php-sieve-manager": "^1.0",
```

Comment:
Intentional?

Reply:
Yes it was intentional. Is it good to lock to 1.0.3 ?

Reply:
It doesn't matter in practice because composer.lock will grab 1.0.3 (soon 1.0.4) but I was just curious :-)